crates/ruff_python_formatter/src/other/f_string_expression_element.rs

```diff
@@ -0,0 +1,16 @@
+use ruff_python_ast::FStringExpressionElement;
+
+use crate::prelude::*;
+
+#[derive(Default)]
+pub struct FormatFStringExpressionElement;
+
+impl FormatNodeRule<FStringExpressionElement> for FormatFStringExpressionElement {
```

Comment:
I'm not sure if it's worth adding these implementations considering that they won't work. I rather have `fstring_expression_element.format()` to not compile than fail at runtime. 

Should we exclude them from the generate script. We can still add them when they become useful.